tree/dataframe/test/dataframe_vary.cxx

```diff
@@ -269,6 +269,32 @@ TEST(RDFVary, GetVariations)
                   "Variations {x:0, x:1} affect column x\nVariations {xy:0, xy:1} affect columns {x, y}\n");
 }
 
+TEST(RDataFrameInterface, VaryFriend)
```

Comment:
I think the first argument should be `RDFVary` but not `RDataFrameInterface`.

If I try MT execution (e.g. provide instead `TEST_P(RDFVary, VaryFriend)`) on this one, I get:
unknown file: Failure
C++ exception with description "The input TTree is not linked to any file, in-memory-only trees are not supported." thrown in the test body.

Is that desired/expected?